change/@itwin-property-grid-react-8367431f-b9d7-46b2-86c8-533d9857b0ae.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "DevDependency update",
+  "packageName": "@itwin/property-grid-react",
+  "email": "43886789+MartynasStrazdas@users.noreply.github.com",
+  "dependentChangeType": "none"
```

Comment:
I'm not sure what type the change should be (none, patch or minor)? @saskliutas @grigasp

Reply:
If there are no api or behavior changes, then it's a patch.